frontend/src/pages/Camera/components/CameraCapture.jsx

```diff
@@ -13,10 +12,9 @@ const videoConstraints = {
   facingMode: "environment",
 };
 
-export default function CameraCapture({ onShowPhotos, onImageCapture }) {
+export default function CameraCapture({ onShowPhotos, onImageCapture, onSubmit}) {
```

Comment:
Really appreciate how extensible the Camera component is now! Great adaptation of the component